x-pack/plugins/ml/server/shared_services/providers/job_service.ts

```diff
@@ -4,19 +4,40 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { APICaller } from 'kibana/server';
-import { LicenseCheck } from '../license_checks';
+import { APICaller, KibanaRequest } from 'kibana/server';
 import { jobServiceProvider } from '../../models/job_service';
+import { SharedServicesChecks } from '../shared_services';
+
+type OrigJobServiceProvider = ReturnType<typeof jobServiceProvider>;
 
 export interface JobServiceProvider {
-  jobServiceProvider(callAsCurrentUser: APICaller): ReturnType<typeof jobServiceProvider>;
+  jobServiceProvider(
+    callAsCurrentUser: APICaller,
+    request: KibanaRequest
+  ): {
+    jobsSummary: OrigJobServiceProvider['jobsSummary'];
+  };
 }
 
-export function getJobServiceProvider(isFullLicense: LicenseCheck): JobServiceProvider {
+export function getJobServiceProvider({
+  isFullLicense,
+  getHasMlCapabilities,
+}: SharedServicesChecks): JobServiceProvider {
   return {
-    jobServiceProvider(callAsCurrentUser: APICaller) {
-      isFullLicense();
-      return jobServiceProvider(callAsCurrentUser);
+    jobServiceProvider(callAsCurrentUser: APICaller, request: KibanaRequest) {
+      // const hasMlCapabilities = getHasMlCapabilities(request);
```

Comment:
Is this commented out line needed?

Reply:
it's related to the commented out code below